packages/account/src/Sections/Security/LoginHistory/__test__/login-history.spec.tsx

```diff
@@ -0,0 +1,170 @@
+import React from 'react';
+import { screen, render, waitFor } from '@testing-library/react';
+import { WS, isMobile } from '@deriv/shared';
+import { StoreProvider, mockStore } from '@deriv/stores';
+import LoginHistory from '../login-history';
+import { getLoginHistoryFormattedData } from '../../../../../../utils/src/getLoginHistoryFormattedData';
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Loading: jest.fn(() => 'mockedLoading'),
+}));
+
+jest.mock('../../../../../../utils/src/getLoginHistoryFormattedData', () => ({
+    getLoginHistoryFormattedData: jest.fn(),
+}));
+
+jest.mock('@deriv/shared', () => ({
+    ...jest.requireActual('@deriv/shared'),
+    isMobile: jest.fn(() => false),
+    WS: {
+        authorized: {
+            fetchLoginHistory: jest.fn(() =>
+                Promise.resolve({
+                    login_history: [
+                        {
+                            date: '2023-08-28 03:11:45 GMT',
+                            browser: 'Chrome  v116.0.0.0',
+                            action: 'login',
+                            status: 'successful',
+                            ip: 'MOCK.IP.ADDRESS',
+                            id: 0,
+                            environment: 'Deriv GO',
+                        },
+                    ],
+                })
+            ),
+        },
+    },
+}));
+
+describe('<LoginHistory />', () => {
+    let mock_store: ReturnType<typeof mockStore>;
+
+    const renderComponent = async () => {
+        const wrapper = ({ children }: { children: JSX.Element }) => (
+            <StoreProvider store={mock_store}>{children}</StoreProvider>
+        );
+        render(<LoginHistory />, { wrapper });
+    };
+
+    beforeEach(() => {
+        (getLoginHistoryFormattedData as jest.Mock).mockReturnValue([
+            {
+                date: '2023-08-28 03:11:45 GMT',
+                browser: 'Chrome  v116.0.0.0',
+                action: 'login',
+                status: 'successful',
+                ip: 'MOCK.IP.ADDRESS',
+                id: 0,
+            },
+        ]);
+        (isMobile as jest.Mock).mockReturnValue(false);
+        mock_store = mockStore({
+            client: {
+                is_switching: false,
+                is_authorize: true,
+            },
+            ui: {
+                is_mobile: false,
+            },
+        });
+    });
+
+    it('should render Login History List when isMobile is true', async () => {
+        mock_store.ui.is_mobile = true;
+        renderComponent();
+        await waitFor(() => {
+            expect(screen.getByText(/date and time/i)).toHaveClass('dc-text login-history__list__row__cell--title');
```

Comment:
*NIT*: Lets try to avoid testing if a class exists (because they may change). Because the testcase is checking if a node is present in DOM. May be there is another way to test based on existence of content or testid?

Reply:
i think here we are intentionally checking if `login-history__list__row__cell--title` is present in mobile view and not present in desktop view. just making sure the right component is rendered based on screen size.